dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -1109,24 +1077,22 @@ function onGameStateChange(msg) {
 
   updateDonationStatus(state.donation_active);
 
-  // Update items
+  // Update gridItems
   if (state.items !== undefined && state.items !== null) {
-    items = [];
+    gridItems = new itemlib.GridItems();
     for (j = 0; j < state.items.length; j++) {
-      var new_item = new Item(
-        state.items[j].id,
-        state.items[j].item_id,
-        state.items[j].position,
-        state.items[j].maturity,
-        state.items[j].remaining_uses,
+
+      gridItems.add(
+        new itemlib.Item(
+          state.items[j].id,
+          state.items[j].item_id,
+          state.items[j].maturity,
+          state.items[j].remaining_uses,
```

Comment:
Seems like we should still be setting the position here, as it might be useful to have as an attribute. See below for a place where it still seems to be expected.